### PR TITLE
[idea] #269 added search with KodeBeagle option to editor right click menu after selecting some code

### LINKED ISSUE #269: Add Search with KodeBeagle before Search with google in editor right click menu

### PR DESCRIPTION
**Pull request by prudhvije**
fixes #269

### CODE REVIEW

**Comment by prudhvije** [comment]
image

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/RefreshAction.java [hunk 1]
```diff
@@ -133,7 +133,7 @@
     private ESUtils esUtils = new ESUtils();
     private JSONUtils jsonUtils = new JSONUtils();
     private PropertiesComponent propertiesComponent = PropertiesComponent.getInstance();
-    private JTabbedPane jTabbedPane;
+    private static JTabbedPane jTabbedPane;
```
Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/RefreshAction.java]: Why is this static ?

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/SearchKodeBeagleAction.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import java.io.IOException;
+
+public class SearchKodeBeagleAction extends AnAction {
+
+    public final void actionPerformed(final AnActionEvent e) {
+        MainWindow mainWindow = new MainWindow();
+        Project project = e.getProject();
+        if (project != null) {
+            ToolWindowManager.getInstance(e.getProject()).
+                    getToolWindow(mainWindow.KODEBEAGLE).show(new Runnable() {
```
Comment (ScrapCodes) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/SearchKodeBeagleAction.java]: You should access static variables directly from `MainWindow.KODEBEAGLE` .

File: plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/RefreshAction.java (hunk 1 above)
Comment (prudhvije) [inline plugins/idea/kodebeagleidea/src/main/java/com/imaginea/kodebeagle/action/RefreshAction.java]: Currently we are never creating the object of RefreshAction but when search with KodeBeagle is hit two things should happen Showing the ToolWindow and calling RefreshAction init method. Without static that instance of jtabbed pane is gone.

**Pull request closed without merging**